ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Bug report or feature request
+about: Describe a bug you've seen or make a case for a new feature
+---
+
+Please briefly describe your problem and what output you expect. If you have a question, please don't use this form. Instead, ask in the Fred Hutch Data Slack <https://join.slack.com/t/fhdata/signup>.
+
```

Comment:
Should we give people the option to email us? (wilds@)

Reply:
Yeah, forgot about that

Reply:
Can you also use this URL so we get the analytics on it? https://hutchdatascience.org/joinslack/

Reply:
oh yeah can do